### PR TITLE
Update domain names

### PR DESCRIPTION
**Pull request by WadeBarnes**
Updates for #696

### CODE REVIEW

**Comment by WadeBarnes** [comment]
Hold off merging this PR.  Waiting on:
- [ ] Some additional DNS entries
- [ ] The certificates

File: tob-web/openshift/angular-on-nginx-deploy.test.param [hunk 1]
```diff
@@ -6,12 +6,12 @@
 # NAME=angular-on-nginx
 # IMAGE_NAMESPACE=devex-von-tools
 # REAL_IP_FROM=172.51.0.0/16
-# API_URL=https://django-devex-von-dev.pathfinder.gov.bc.ca/api/V2/
+# API_URL=https://dev-demo-api.orgbook.gov.bc.ca/api/V2/
```
Comment (esune) [inline tob-web/openshift/angular-on-nginx-deploy.test.param]: Should this be `demo-api.orgbook.gov.bc.ca/api/V2/`, since it is for the `test` environment?

Comment (WadeBarnes) [reply tob-web/openshift/angular-on-nginx-deploy.test.param]: yes

Comment (WadeBarnes) [reply tob-web/openshift/angular-on-nginx-deploy.test.param]: updated

**Comment by WadeBarnes** [comment]
Leaving the prod configurations for the moment.

**Pull request merged**